requirements.txt

```diff
@@ -15,19 +15,24 @@ kaleido # for exporting plotly images to png / html
 # embeddings:
 openai==0.28 # ok so there is now a 1.0 interface but we originally obtained using earlier API, so pinning that here
 
+#google cloud:
+google.cloud
+pandas-gbq
+google-cloud-bigquery
+
 
 # machine learning:
 scikit-learn #==1.3.0
 #category_encoders
 #tensorflow
-xgboost==1.7.6 #==1.2.1 #==1.5.0 # 1.7.6
+#xgboost==1.7.6 #==1.2.1 #==1.5.0 # 1.7.6
```

Comment:
let's keep these uncommented as they were, and if you need to abbreviate which packages to install locally, you can temp. comment out and then restore the file.